cmd/minikube/cmd/logs.go

```diff
@@ -97,13 +96,7 @@ var logsCmd = &cobra.Command{
 			logs.OutputProblems(problems, numberOfProblems, logOutput)
 			return
 		}
-		err = logs.Output(cr, bs, *co.Config, co.CP.Runner, numberOfLines, logOutput)
-		if err != nil {
-			out.Ln("")
-			// Avoid exit.Error, since it outputs the issue URL
-			out.WarningT("{{.error}}", out.V{"error": err})
-			os.Exit(reason.ExSvcError)
-		}
+		logs.Output(cr, bs, *co.Config, co.CP.Runner, numberOfLines, logOutput)
```

Comment:
it is not a good idea to ignore error,

how about we check for error and if it is becaue we cant get logs because it is not running then just tell the out.Warning("cant get logs for ...") without exiting

Reply:
Added it back